simulation/traffic_simulator/src/hdmap_utils/hdmap_utils.cpp

```diff
@@ -828,32 +831,44 @@ auto HdMapUtils::getFollowingLanelets(
   const lanelet::Id lanelet_id, const lanelet::Ids & candidate_lanelet_ids, const double distance,
   const bool include_self) const -> lanelet::Ids
 {
+  lanelet::Ids ids;
+  lanelet::Id previous_id;
+
   if (candidate_lanelet_ids.empty()) {
-    return {};
+    return ids;
   }
-  lanelet::Ids ids;
+
+  bool isNextLanelet{false};
```

Comment:
move this variable down, where it is used

Reply:
variable removed